variables.tf

```diff
@@ -8,6 +8,11 @@ variable "create_asg" {
   default     = true
 }
 
+variable "create_before_destroy" {
```

Comment:
I put this near the top of the options list, I can move it if you like. I also set the default to be the same as current.

I personally think the default should change to `true` :)